CONTRIBUTING.md

```diff
@@ -2,180 +2,10 @@
 
 Welcome to the `jupyter-book` repository! We're excited you're here and want to contribute.
 
-These guidelines are designed to make it as easy as possible to get involved.
-If you have any questions that aren't discussed below, please let us know by opening an [issue][link_issues]!
+The Jupyter Book project is run by a community of people like you, we'd love to have you
+help out!
 
-Before you start you'll need to set up a free [GitHub][link_github] account and sign in.
-Here are some [instructions][link_signupinstructions].
+Please take a look at the [Jupyter Book contributor guide](https://jupyterbook.org/contributing)
+which steps you through the codebase and how to contribute to this project.
```

Comment:
Is it worthwhile to still say something here like :

`If you have any questions that aren't answered there, please let us know by opening an [issue][link_issues]!`

Reply:
yep, that sounds good - though I'd make it `[by open an...` (so move the open-square brackets a few words earlier)